packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -106,7 +107,7 @@ export class RunningSummarizer implements IDisposable {
         private readonly summaryCollection: SummaryCollection,
         private readonly cancellationToken: ISummaryCancellationToken,
         private readonly stopSummarizerCallback: (reason: SummarizerStopReason) => void,
-        { disableHeuristics = false }: Readonly<Partial<ISummarizerOptions>> = {},
+        private readonly disableHeuristics: boolean = false,
```

Comment:
Can we get rid of this argument altogether? Tere is this.configuration that can be used for that.

Reply:
We can't as there is a call to the summarizer (summarizeOnDemand)  in which we enforce the disableHeuristics to be true independent of the configuration.

Reply:
I personally think it is more confusing to have a new config  but I don't have a strong objection.